registration.js

```diff
@@ -0,0 +1,71 @@
+/*
+document.getElementById('form_id').addEventListener("submit", function(event){
+    event.preventDefault();
+    return validate();
+});
+*/
```

Comment:
Try do not leave commented code block in any project.
Git alvays keep in mind all your changes, so it don't necessary and make code more readable.